### ISSUE TITLE
Problem with Page about Date

### ISSUE THREAD

**Issue by aletrovato** [OfficeDev/office-scripts-docs#476]
I would like to report a problem with this page. 

Suggestion 1:
As I am looking for information on working with dates in office script, the text has a problem with the link in this paragraph:
"The next sample reads a date that's stored in Excel and translates it to a JavaScript Date object. It uses the date's numeric serial number as input for the JavaScript Date."
The link is directing to the Now function page.

Suggestion 2
The script shown can be simplified to the code below, because in this snippet of the original ( let excelDateValue = dateRange.getValue() as number;) returns an error.

function main(workbook: ExcelScript.Workbook) {
  // Read a date at cell A1 from Excel.
  let dateRange = workbook.getActiveWorksheet().getRange("A1").getValue();

  // Convert the Excel date to a JavaScript Date object.
  let javaScriptDate = new Date(Math.round((dateRange - 25569) * 86400 * 1000));
  console.log(javaScriptDate);
}

Thank you!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 98c58cbf-aaf7-3d3a-48f4-ce9cebcf331c
* Version Independent ID: 74821d34-eab6-ec0f-7a9f-55aca81975d9
* Content: [Basic scripts for Office Scripts in Excel on the web - Office Scripts](https://docs.microsoft.com/en-us/office/dev/scripts/resources/samples/excel-samples#dates)
* Content Source: [docs/resources/samples/excel-samples.md](https://github.com/OfficeDev/office-scripts-docs/blob/main/docs/resources/samples/excel-samples.md)
* Product: **excel**
* Technology: **scripts**
* GitHub Login: @o365devx
* Microsoft Alias: **o365devx**

**Comment by AlexJerabek** [comment]
Thanks for pointing out this issue @aletrovato. I'll get the page updated soon.

**Comment by aletrovato** [comment]
How amazing the speed of the response.
Thank you so much Alex!!!



*Alessandro Trovato*

*Microsoft MVP / Instrutor LinkedIn Learning / Youtuber*



Blog, Catálogo Virtual das Aulas e cursos com certificado
<https://linktr.ee/alessandrotrovato>

Acesso ao Youtube <http://www.youtube.com/aletrovato>

App gratuito na Play Store <https://bit.ly/apptrovato> com as aulas


*Compartilhando conhecimento, sempre!*




Em qui., 24 de mar. de 2022 às 13:58, Alex Jerabek ***@***.***>
escreveu:

**Comment by AlexJerabek** [comment]
Hi @aletrovato,

First, thank you for raising this issue. I have reached out to the product team to see about adding convenience methods to Office Scripts to make working with dates easier and more intuitive.

I wanted to follow up about your two suggestions. For the NOW function link, I had originally linked to that page because it contained the best description of the Excel date serial number. Is that link confusing? Should we just remove it?

For the sample not working, I could not reproduce that issue. Could you please provide a screenshot of the error or a copy of the text that is produced?

Thank you,

**Comment by aletrovato** [comment]
Hi Alex!

Your idea to develop a more practical way to work with dates is very welcome. When I tested today some scripts for date handling I ran into the problem of using the Date library (I didn't know it yet). The conversion of DataSerial types to standard JavaScript is confusing, perhaps it could also be an item in the Office Script help pages. I have written articles, produced a course and several classes for this on Youtube. I am trying to raise the flag and train users here to migrate to WEB and it would be of great help.

Suggestion 1 NOW - The redirect got confusing, if I may suggest, it would be interesting to remove the link or include the information that the NOW page illustrates how the date will look after formatting.

Suggestion 2: Solved. In the morning, when testing the code, getValue() was not recognized as a valid statement. 
If I can help with anything, I am at your disposal.

Thank you very much!

![image](https://user-images.githubusercontent.com/12089311/159995483-a8d65123-e34e-41f9-bf3f-10555bc7fc39.png)
